src/backoffice/settings/logviewer/workspace/logviewer.context.ts

```diff
@@ -101,18 +104,7 @@ export class UmbLogViewerWorkspaceContext {
 		this.#repository = new UmbLogViewerRepository(this.#host);
 	}
 
-	async init() {
-		this.validateLogSize();
-	}
-
-	reset() {
-		this.#logs.next({ items: [], total: 0 });
-		this.setCurrentPage(1);
-	}
-
 	onChangeState = () => {
-		this.reset();
```

Comment:
I thought it made sense to at least clear the visible logs here when searching for new logs to be sure that if a server errors occurs, then we don't show out-dated logs. What do you think about that?

Reply:
There is a loading state implemented to "cover" that moment :) With the reset it was bit to flickering. It look slike this now, so there is no risk in seeing "old" logs from before the request 
![log-viewer-loading staTE](https://user-images.githubusercontent.com/56249914/235863304-85d4a4b9-384b-4064-9a8b-03d8ad489ba6.gif)